monkey/tests/unit_tests/agent_plugins/credentials_collectors/mimikatz/test_plugin.py

```diff
@@ -203,6 +203,6 @@ def test_exclude_username_prefixes(monkeypatch):
     }
     options = MimikatzOptions(excluded_username_prefixes=["sensitive-", "admin"])
 
-    collected_credentials = collect_credentials(options.dict(simplify=True))
+    collected_credentials = collect_credentials(options.model_dump(mode="json"))
```

Comment:
Should we add a convenience method to `InfectionMonkeyBaseModel` that does this? Maybe `to_json_dict()` or  some such thing?